src/script.js

```diff
@@ -0,0 +1,346 @@
+
+    if (AllCodesAndKeys[event.code][0] === 'CapsLock') {
+      if (document.getElementById(`${event.code}`).classList.contains('active')) {
+        document.getElementById(`${event.code}`).classList.remove('active');
+      } else {
+        document.getElementById(`${event.code}`).classList.add('active');
+      }
+    } else {
+      document.getElementById(`${event.code}`).classList.add('active');
+    }
+
+    if (document.getElementById('ShiftLeft').classList.contains('active') && document.getElementById('ControlLeft').classList.contains('active')) {
+      this.changeLanguage();
+      return;
+    }
+
+    if (event.code.match(/(Arrow).*/)) {
+      this.printedText.value += document.getElementById(`${event.code}`).innerHTML;
+    }
+
+    if (this.language === 'EN' && AllCodesAndKeys[event.code][1] !== 'controlButton') {
+      if (this.shiftStatus === false && this.CapsLockStatus === false) {
+        this.printedText.value += AllCodesAndKeys[event.code][0];
+      } else if (this.shiftStatus === true && this.CapsLockStatus === false) {
+        this.printedText.value += AllCodesAndKeys[event.code][1];
+      } else if (this.shiftStatus === false && this.CapsLockStatus === true) {
+        if (!event.code.match(/(Digit)[0-9]/) && !event.code.match(/.*(ash)/) && event.code !== 'Backquote') {
+          this.printedText.value += AllCodesAndKeys[event.code][1];
+        } else {
+          this.printedText.value += AllCodesAndKeys[event.code][0];
+        }
+      } else if (this.shiftStatus === true && this.CapsLockStatus === true) {
+        if (!event.code.match(/(Digit)[0-9]/) && !event.code.match(/.*(ash)/) && event.code !== 'Backquote') {
+          this.printedText.value += AllCodesAndKeys[event.code][0];
+        } else {
+          this.printedText.value += AllCodesAndKeys[event.code][1];
+        }
+      }
+    } else if (this.language === 'RU' && AllCodesAndKeys[event.code][1] !== 'controlButton') {
+      if (this.shiftStatus === false && this.CapsLockStatus === false) {
+        this.printedText.value += AllCodesAndKeys[event.code][2];
+      } else if (this.shiftStatus === true && this.CapsLockStatus === false) {
+        this.printedText.value += AllCodesAndKeys[event.code][3];
+      } else if (this.shiftStatus === false && this.CapsLockStatus === true) {
+        if (!event.code.match(/(Digit)[0-9]/) && !event.code.match(/.*(ash)/)) {
+          this.printedText.value += AllCodesAndKeys[event.code][3];
+        } else {
+          this.printedText.value += AllCodesAndKeys[event.code][2];
+        }
+      } else if (this.shiftStatus === true && this.CapsLockStatus === true) {
+        if (!event.code.match(/(Digit)[0-9]/) && !event.code.match(/.*(ash)/)) {
+          this.printedText.value += AllCodesAndKeys[event.code][2];
+        } else {
+          this.printedText.value += AllCodesAndKeys[event.code][3];
+        }
+      }
+    } else if (event.code === 'CapsLock') {
+      if (this.CapsLockStatus === false) {
```

Comment:
`if (!this.CapsLockStatus) ...` for shortand

Reply:
fixed